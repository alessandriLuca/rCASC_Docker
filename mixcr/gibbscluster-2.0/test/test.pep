TMCIYQSPVRKLKLAC
FKLALFFIIFNKQFHQI
RPPIFIRRLPSCHN
CTVHQLPAIIRPSFW
RARIKTRLF
TDKLYGLSQGY
SDCDQMVIENGILKNK
RGRIYKRSLKLDDLLS
IKRNAVLIDAGLPKLQ
IILAALFMYYTKYW
AVLQSGFRK
DLKVLSHGWAYKA
YVVMMVIFRLMRDT
GYHNSPASFFSSWVQA
SGTNIILAHFLYKEFTMN
KKTVLEFILQKIFKH
VNSPADERAVANFW
QIPAFYTPTARV
MWEPGLIFCHSKKKD
PCYKPESRPFDL
DIKDAVYSICKFY
NHCRPRQIAMSLFIM
EMQHRFPIAKVEPVQKQK
PIDPYVVMQVKVPKIRIF
FRDHGATSIYTIERAPMFW
EKKDFRAYRNALSMPIRMN
VPHISRQRLPWL
LLPTNAVVKMKK
FFVACAPFMSDLQFPMTWH
GDGGTRVAVNKSNKQRHS
RHLTPKFTRMVVSN
WHPAQLFKPLTKKFMD
ANHVGALKISQLQKNFPP
FESMTLVPVLEKALMPK
TRPLRMPAYIRNTLYQA
GTEDAPKRPPSAF
IWYPKGPIRFVLAL
NWCIPKRNRSILP
KLRETGAPLP
CMYTRVYRRRLAQS
HEPVYPGIKVRQLT
FTEAQPKPGTRMVPCT
EWCETLSERISSKRYGRA
PALYSYASAKWW
MENSTMTIREFPRKSYVM
IAPLLSAGIFH
LKVSWRWMVY
ILLFYCPQKKKSQAEFWD
EINEWLSSKDRR
INSNDFPVRPQVPITRCCC
LFRMKAYKIISLKEVEYF
LYDSFTYLCGFIKSLCC
FAGALIRATSTRIGDG
GLSPSKRNQTFIPW
GLPTRCRLEIVHGMG
KFVNSPLQLRFRMII
AENKLYKMRIPRHL
RTLHPFGCKSN
QCRFPASFFIKLFPFM
PERTFGKLPYRNL
YIDQMCVFKFIVAKS
GVFKMGAFMYTKTYD
HEALDGTFQRKYMAF
QRFYPIVIAPAMNEDSH
QRHDTLARSICEK
TALMRWRHPRVVSDS
MYMVYVPHVIEEVMRLSL
KYLYLWSFKPTMQ
RTMSGLPAIVREAIEE
HLVNAISRLRTQKANSIC
SQIQAWPLLPHVIF
EIYALLAVGATKGA
SAIFASSMTKTN
DLFFWYTNEIIGYKVAL
LWRRTKLMSNIKMNC
HQPHRVMANNVKKDIII
KLAQVRRAMCAD
VAMSTWSFYH
RQFVSNNGKVPH
ESYSLPRPDTRHLHVWLN
CIMFEQYFIYW
FCHNISFKSINKQ
IEITPFGQQRVFYIYW
SMEWTLWKAGILYDTP
DCVHHSPGEIKPKFDEGW
WAPSPYNSRFCAGW
EYRVRSLAKSLHW
ASFIAGMLVGRLGKGY
KEDLAWQRTLKLS
VRWGVTLYVKALTKHFTS
GWSDLPPFTQHLLG
YPSLMSRVVAVM
MPTDGLVGFCF
ECCGVPWSKILAYNADPI
VLKERLIEKTNEKDQKNK
NELPNVDLTTMM
TQAMRAPKVRLSLW
ATRIPLGGNGAM
CLKDIIGPMFSGKVLDE
HIHIHPLARTAKVAQSYC
EWTPVMSRFAAWIIPE
LHDIVRTNRNELLLMK
RPVPISHLYIL
KPNELSLALSEG
IWLKLAIFQASMTK
SFLPRRSLKAFDSK
RHTVNPIVTEKY
YIFFASFYY
FPRGQGVPI
MRPFNNILNLSQ
SFIDLIVFNLPVSKM
LAGLPYPQPQPFML
YQEMLSRVAAVKFQQ
DFDNQRPNMSRHHF
MKSMNLPEFERRTLQ
REFPSIFSTEVHSVAG
CDAIITPVVFYRWM
LEHPIMYYTKFMLFT
LAIFQSAMTK
FMKLAETRMGYMR
TYKCLAVFKDSFLRG
CFRLSLYGRYNCKC
HMSGVAMPNLYKLIYH
CPIGENSPLQK
VISPEQTQTYNIGKF
CSSVITQACPKFFVGA
YSLHIVKYKQYLKSF
RKGSMLPQIGGEAIKY
HIRLHQQALGCAQ
VAVCGLPQYFTFDLKC
EWSRTGTRLLGRKRMTS
YNSYGVYSVFYLYLCLT
ALFGQALHKAGD
QNPANKEESIHCI
SLPRRWFFRPLRKPY
GSYGPATAQMALPDY
DFVMTYSHHACRNYN
AHRDCLPTWLGAAIEW
FPKYTMKPHTFNKCY
VFIGFPPGTSLTISD
AFGRTLNLFRYKW
YQIRTFNEDLFRD
MHLAAAQGQAPL
HPSDGKCNLN
TCIPTFLQEALYHKF
LNWTVWNRLIARKLQWT
WCDQLPLMMLSPLLDK
NEGCQLTYRHKVVKS
NEKRLPAGSRIWLGPLA
GWRNQPAATALT
ASFDKAKLKQK
VEKLSSMAAERRG
FWNAKPLKDVERLT
PAIPSIQSRGLI
GRSIAMLKSKRDALQ
YMFESKSMKTH
RPNRQLGSMHTSKY
KNQNGVYPSFMSRPH
MKLRKPKHKKSFK
PAAYYFMKFRYL
RPNITSTALSV
SPRWYFYYLWCF
SIIPSGPLKIDMW
AHWNYFPAPRAETL
ICFRVYINVVVKLFYE
WIYDPRNQKTDT
IMQFKLGIPK
APALATGSF
LVPKLMEEYLRRMDSVK
IMQDWYPKLKNSYVRA
DDEDYAPIMFSNKMC
KTKPFRMVSLVGCWNA
WRDIPFSEGKALWKN
WQGIKLRHGQRSLF
EDFLPVNVAFELAPKW
IMGFGHRIYKCYSRF
KKMLEASTIYRS
TRLYYDSMSY
CKVFFGPIYY
IKRQGSTPLWNQA
ALANMTDAIRTLKKH
PTVACPKVSFEPIYSM
VMADPIPRRNVATL
RIYCQENPYNNVD
LRPQASGVYMSADT
GYKLVTSSGAIYKIYWM
MWFTFPVRPQVPHTG
GMSSRAIAR
QRIRSPRTLNAWVNYVAT
RERLSRMAIK
SYILWKDIFHK
SPNMSLLRDNRAALL
NWPALSSIAATKP
GKAANEIRISKNRGPT
MPREDAHFI
FFPPTSFGPLPAWD
QVGTGVSENIFLKWPP
HPAAMPHLLN
ARSRLVSLALHKDN
SYEPLIWAYLSKKR
