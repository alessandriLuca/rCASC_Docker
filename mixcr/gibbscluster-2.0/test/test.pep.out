#Call: /Users/massimo/Applications/gibbscluster-2.0_DB/gibbscluster-2.0/GibbsCluster-2.0e_SA.pl -f test.pep -P gibbs
#Mon Apr  3 09:34:42 2017
#Session ID: 4486
#Run name: gibbs_4486
#Read 200 unique sequences
##### Settings: #####
#Shift moves and cluster moves are activated
#Number of clusters: 2
#Motif length: 9
#Initial MC temperture: 0.8
#Number of temperature steps: 20
#Number of iterations x Sequence x Tstep: 100
#Max insertion length: 0
#Max deletion length: 0
#Interval between Single Peptide moves: 20
#Interval between Phase Shift moves: 100
#Number of initial seeds: 1
#Penalty lambda: 0.8
#Weight on small clusters: 5
#Preference for hydrophobic P1: 0
#Sequence weighting type: 0
#Use trash cluster to remove outliers: 0
#### Running Gibbs clustering...
#Clustering with 2 groups, seed number 1
#Clustering complete!
### Determining seeds with highest KLD...
#Best 2 groups, seed 1
###Parsing result files...
#RESULTS for 2 CLUSTERS
#2 Final Average KLD: 7.403822
#2	1	96	7.100
#2	2	104	7.685
